R/get_coverage_wcs.R

```diff
@@ -21,7 +21,8 @@
 #'   - `"dtm"`: digital terrain model Flanders
 #'   - `"dsm"`: digital surface model Flanders
 #'   - `"dhmv"`: digital elevation model Flanders (contains dtm and dsm data)
-#' See [metadata Vlaanderen](https://metadata.vlaanderen.be/srv/eng/catalog.search#/search?any=WCS) for more information # nolint: line_length_linter.
+#' For more information, see metadata Vlaanderen:
+#'   https://metadata.vlaanderen.be/srv/eng/catalog.search#/search?any=WCS
```

Comment:
Two remarks here (see https://inbo.github.io/inbospatial/reference/get_coverage_wcs.html):

- markdown requires a whiteline for a new paragraph
- the hyperlink is not clickable. I expect it will work when wrapping in `<` `>`